network/peers/connections/testing.go

```diff
@@ -0,0 +1,58 @@
+package connections
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"time"
+
+	"github.com/bloxapp/ssv/network/records"
+	"github.com/bloxapp/ssv/utils/rsaencryption"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+var (
+	SenderPrivateKey *rsa.PrivateKey
+
+	HandshakeData       records.HandshakeData
+	HashedHandshakeData []byte
+	Signature           []byte
+
+	SenderPeerID    = peer.ID("1.1.1.1")
+	RecipientPeerID = peer.ID("2.2.2.2")
+
+	PrivateKeyPEM      []byte
+	SenderPublicKeyPEM []byte
+)
+
+func prepareTestingData() {
```

Comment:
this and all below is still a global state. tests in go run in parallel meaning multiple tests will run this function and reset the values. this should be more of a contained function that creates the variables for each test.

also instead of panicking you can pass in `testing.T` and call `Error`.

Reply:
awesome idea! I will do!